app/survey/utils.tsx

```diff
@@ -0,0 +1,369 @@
+import { AgeResults, 
+    AffordFairholdResults, 
+    CountryResults,
+    HousingOutcomesResults,
+    SankeyResults,
+    SurveyResults, 
+    SupportDevelopmentResults,
+    SupportDevelopmentFactorsResults,
+    SupportNewFairholdResults,
+    FormattedSurveyResults, 
+} from "./types";
+
+export const formatSurveyResults = (results: SurveyResults[]): FormattedSurveyResults => {
```

Comment:
nit: We might be better thinking of these as raw results and results. We could end up passing around the prefix `Formatted` a whole lot when contextually it's not that relevant to the front end (which never has a distinction of raw vs formatted).

```suggestion
export const formatSurveyResults = (results: RawSurveyResults[]): SurveyResults => {
```

```suggestion
export const formatSurveyResults = (results: AirtableSurveyResults[]): SurveyResults => {
```

Reply:
Good call, pattern followed in #485 which should replace this PR!